### ISSUE TITLE
Show successful projects on the monitor page

### ISSUE THREAD

**Issue by cisum-la** [build-canaries/nevergreen#220]
Feature Request:

Add a button in NeverGreen Settings page to display all pipeline statuses (not just green)

I understand this probably goes against "NeverGreen" name, but practically in our situation we are getting requests to see the "Green" pipelines as well, So it will be great if this request can be done.

Our workflow is we select specific pipelines from multiple "GoCD" servers and then monitor it's status (Green, Red, Orange etc.) which includes Green status as well.

**Comment by GentlemanHal** [comment]
Hi @cisum-la, thanks for raising this feature request.

This is certainly against one of the core principals of Nevergreen. That's not to say we won't ever implement such a feature, it just isn't going to be high priority.

**Comment by cisum-la** [comment]
I am thinking of working on this, which IDE would you recommend. Will Visual Studio Code work- I see react for frontend and Clojure for backend and the Cctray parsing.

**Comment by GentlemanHal** [comment]
Personally I use [IntelliJ IDEA Ultimate](https://www.jetbrains.com/idea/) with the [Cursive](https://cursive-ide.com/) plugin.

**Comment by cisum-la** [comment]
Thanks @GentlemanHal  for above info.

**Comment by GentlemanHal** [comment]
I decided this feature was no longer low priority after finding out Jenkins pipelines with manual approval steps remain building until the pipeline completes. Meaning if you have a manual approval step before deploying to production for example, your pipelines could be marked as building for days depending on your release cycle!

No other CI server I've come across behaves like this, they mark projects waiting on manual approval as sleeping e.g. GoCD and CircleCI. 

Rather than just allowing healthy builds I've given the user full control over which prognoses they want to show on the Monitor page. Which means they could hide building projects, which would at least make Nevergreen useful for giving broken build feedback when using Jenkins pipelines with manual steps...
